.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+**Is this pull request related to an existing issue? What is the issue number?**
+
+E.g. #47 (Creation of docs folder / corresponding docs site)
+
+**Description of proposed changes:**
+
+E.g. "This pull request implements a docs folder that provides documentation for the natlas project."
+
+**Should anyone specifically look at this?**
+
+E.g. "@0xdade should probably review this because he cares a lot about the documentation of the project."
+
+**Has this contribution gone through any independent testing?**
+
+E.g. "I tested these things, wrote these tests, and these were the results"
+
+**Does this contribution meet the guidelines recommended in [CONTRIBUTING.md](CONTRIBUTING.md)
+
+No/Yes
+
+**Any other necessary information or context for this contribution?**
```

Comment:
In the other sections there is an opt out or an example. I would like this to say 

E.g. N/A

as opposed to nothing to either A. force the individual issuing the PR to say something or B. keep it N/A.